tests/test_patch_flask_user.py

```diff
@@ -6,32 +6,32 @@
 class TestPathFlaskUser(TestCase):
```

Comment:
Does this need to inherit from `TestCase`?

Reply:
Great point - too bad I didn't think about that or see this prior to merge.  This is probably a good next step for @John0Randolph as this test class can be easily ported to pytest.